pkg/injector/pod_patch.go

```diff
@@ -333,6 +338,30 @@ func getSidecarContainer(applicationPort, applicationProtocol, id, config, daprS
 			"--sentry-address", sentryAddress,
 			"--metrics-port", fmt.Sprintf("%v", metricsPort),
 		},
+		ReadinessProbe: &corev1.Probe{
+			Handler: corev1.Handler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Path: sidecarHealthzPath,
+					Port: intstr.IntOrString{IntVal: sidecarHTTPPort},
+				},
+			},
+			InitialDelaySeconds: defaultHealthzProbeDelay,
+			TimeoutSeconds:      defaultHealthzProbeTimeout,
+			PeriodSeconds:       2 * defaultHealthzProbeTimeout,
```

Comment:
Why do we need to multiply default by 2 ?

Reply:
I tried to align this defaults to K8s defaults.